akvo/templates/project_directory.html

```diff
@@ -264,6 +271,26 @@ <h4 class="detailedInfo">{% trans "Finance" %}</h4>
         }
     </script>
 
+    {# More partners data #}
+    {% for project in page %}
+    <script type="application/json" id="more-partners-{{project.id}}">
+    [
+    {% for partnership in project.partnerships.distinct %}
```

Comment:
You are right about `.distinct` destroying the prefetch, and it looks like we don't really need a distinct here. Good catch! Thanks.